app/styles/libs/_helpers.scss

```diff
@@ -136,10 +136,10 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 }
 
 .social-link-button {
-  width: 50px;
-  height: 50px;
-  max-width: 50px;
-  max-height: 50px;
+  width: 59px;
+  height: 60px;
+  max-width: 59px;
+  max-height: 60px;
```

Comment:
why?

Reply:
Sir Actually in this issue, Width of those buttons had to increased, after doing that buttons got asymmetrical ,i.e. 5 buttons in 1 and 2 row and 2 buttons in last row which was not looking good, So I increased their dimensions to again make them symmetrical.